po/R-zh_CN.po

```diff
@@ -559,83 +563,103 @@ msgid ""
 "copying: ?set, ?setnames and ?setattr. If this message doesn't help, please "
 "report your use case to the data.table issue tracker so the root cause can "
 "be fixed or this message improved."
-msgstr ""
+msgstr "侦测到无效的 .internal.selfref ，已藉由获取 data.table 的(浅层)副本的方式修复"
+"以便 := 能利用引用加入新列"
+"在过去的某个时间点，此 data.table 已被 R 复制(或是以 structure()等方式手动生成)"
+"请避免 names<- 与 attr<- 等目前(与偶发)可能会在 R 中复制整个data.table的操作"
+"请改用 set* 语法以避免复制，详见 ?set、?setnames 及 ?setattr"
+"如果以上讯息无法提供帮助，请回报你的案例至 data.table 问题追踪"
+"以助于修复根本原因或改进本讯息"
 
 msgid ""
 "Cannot assign to an under-allocated recursively indexed list -- L[[i]][,:=] "
 "syntax is only valid when i is length 1, but it's length"
 msgstr ""
+"无法指定配置不足的递归索引列表"
+"-- L[[i]][,:=] 语法只有在 i 长度为1时有效，但它的長度"
 
 msgid "Internal error -- item '"
-msgstr ""
+msgstr "内部错误 -- 项目"
 
 msgid "' not found in names of list"
-msgstr ""
+msgstr "未能在名称列表中找到"
 
 msgid "Internal error -- column(s) not found:"
-msgstr ""
+msgstr "内部错误 -- 找不到此列:"
 
 msgid ""
 "strptime() usage detected and wrapped with as.POSIXct(). This is to minimize "
 "the chance of assigning POSIXlt columns, which use 40+ bytes to store one "
 "date (versus 8 for POSIXct). Use as.POSIXct() (which will call strptime() as "
 "needed internally) to avoid this warning."
-msgstr ""
+msgstr "侦测到使用 strptime() ，已用 as.POSIXct() 包裹"
+"这么做是为了尽量避免列被指定为 POSIXlt"
+"POSIXlt用40个以上的位元组储存日期(相较于 POSIXct 只用8位元组)"
+"请使用 as.POSIXct() 以避免本警告 (此函数会根据需求在内部调用 strptime())"
 
 msgid ""
 "' is not found in calling scope. Looking in calling scope because this "
 "symbol was prefixed with .. in the j= parameter."
 msgstr ""
+"未能在 calling scope 中找到，请查看 calling scope"
+"因为  j= 的参数以 .. 作为前缀"
 
 msgid "Internal error: xcolAns does not pass checks:"
-msgstr ""
+msgstr "内部错误 : xcolAns 无法通过检查:"
 
 msgid ""
 "Internal error: irows is NULL when making join result at R level. Should no "
 "longer happen now we use CsubsetDT earlier."
-msgstr ""
+msgstr "内部错误 : 在 R 生成连接结果时，irows为 NULL "
+"我们已使用了 CsubsetDT，现在不应该再发生了"
 
 msgid "j (the 2nd argument inside [...]) is a single symbol but column name '"
-msgstr ""
+msgstr "j ( [...] 中的第二顺位参数) 是单个符号而非列名"
 
 msgid "' is not found. Perhaps you intended DT[, .."
-msgstr ""
+msgstr "未被找到，也许你打算 DT[, .."
 
 msgid ""
 "]. This difference to data.frame is deliberate and explained in FAQ 1.1."
 msgstr ""
+"] ，在常见问题 1.1 中有解释dat.table与data.frame的差别"
```

Comment:
maybe not translate `FAQ`? I am not sure

Reply:
agree, FAQ should be kept here